rhea/build/boards/altera/_de0nano_soc.py

```diff
@@ -29,15 +33,113 @@ class DE0NanoSOC(FPGA):
         'led': dict(pins=('W15', 'AA24', 'V16', 'V15',
                           'AF26', 'AE26', 'Y16', 'AA23',)),
         'key': dict(pins=('AH17')),
-        'sw': dict(pins=('L10', 'L9', 'H5', 'H6')),
+        'sw': dict(pins=('L10', 'L9', 'H6', 'H5')),
         
-        # bi-directional GPIO, naming slightly different than the
-        # reference manual and schematics.  The GPIO are not separated
-        # into GPIO_0 and GPIO_1
-        # @todo: finish the GPIO pins
-        'gpio': dict(pins=('V12', 'AF7', 'W12', 'AF8', 'Y8', 'AB4',
-                           'W8', 'Y4', 'Y5', 'U11', 'T8', 'T12')),
-    }
+        # bi-directional GPIO, 
+        # @todo: finish the GPIO pins  (Completed)
+
+        'gpio_0': dict(pins=(	'V12', # GPIO_0[0]
```

Comment:
Don't break these out into `gpio_0` and `gpio_1`, stay consistent between de0nano, de1soc (both those simply have `gpio`).

Also format them in groups as done in the `de0nano` the pin definitions don't take up so many lines.